appcues/src/main/java/com/appcues/ui/extensions/ModifierExt.kt

```diff
@@ -133,6 +133,7 @@ internal fun Modifier.styleShadow(style: ComponentStyle?, isDark: Boolean): Modi
         if (style?.shadow != null) Modifier.coloredShadow(
             color = style.shadow.color.getColor(isDark),
             radius = style.shadow.radius.dp,
+            cornerRadius = style.cornerRadius.dp,
```

Comment:
double checking - this is always non-null, defaults to 0dp corner radius right?

Reply:
![image](https://user-images.githubusercontent.com/5244805/178542329-02a84317-d574-40bf-b15e-9c7e8e660f36.png)
yes